internal/keycloakb/pii.go

```diff
@@ -0,0 +1,30 @@
+package keycloakb
+
+import (
+	"github.com/cloudtrust/keycloak-bridge/internal/constants"
+	"github.com/cloudtrust/keycloak-client"
+	kc "github.com/cloudtrust/keycloak-client"
+)
+
+var (
+	// These attributes should not be used anymore
+	attrbBirthDateLegacy = kc.AttributeKey("birthDate")
+	attrbGenderLegacy    = kc.AttributeKey("gender")
+
+	piiAttributes = map[kc.AttributeKey]kc.AttributeKey{
+		constants.AttrbBirthDate: attrbBirthDateLegacy,
+		constants.AttrbGender:    attrbGenderLegacy,
+	}
+)
+
+// CheckPII verify that PII are located in the well named attributes
+func CheckPII(user *keycloak.UserRepresentation) {
```

Comment:
ConvertLegacyPIIIfAny() ? 
Check could let think this func do not perform any modification on the provided UserRep ?